content/getting-started/finding-kata.md

```diff
@@ -6,8 +6,42 @@ next: "/getting-started/solving-kata/"
 
 # Finding Kata
 
-<!---
-TODO training suggestions
-TODO kata search page
-TODO kata "numbers": rank, satisfaction, possibly others (which?) as criteria for selecting a kata to solve
--->
+Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.
+
+## Codewars trainer
+
+The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.
+
+![training routines](./img/finding-kata_01_training-routines.png)
```

Comment:
Can you take new screenshots so it shows the current UI? I'll update the popup too.

Reply:
I updated screenshots but it turns out I have no proper image editing software installed on my office laptop, so I am not really satisfied with the result.
Any help here would be welcome.

Reply:
We can improve it later, but here's my take:
![sidebar](https://user-images.githubusercontent.com/639336/87838157-e1398600-c84a-11ea-8091-0ce93ff4a685.png)
![training-routines](https://user-images.githubusercontent.com/639336/87837996-56f12200-c84a-11ea-95b9-ecc8089d8dee.png)

We might need to retake the screenshot and make sure it's not too wide. I'll look too short on our docs.